# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: layout/base/crashtests/crashtests.list
# Commit: 9fa579e2d7b9
# Full Hash: 9fa579e2d7b97c7e6eaf77461d40fe94c06ba8d6
# Author: David Shin <dshin@mozilla.com>
# Date: 2024-11-21 04:47:28
# Description:
#   Bug 1931933: Use CSSMinMax instead of std::clamp in nsTextControlFrame::GetNaturalBaselineBOffset. r=layout-reviewers,emilio
#   
#   There is no guarantee that the block size is positive, since the size can overflow.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D229678
# ==============================================================================

diff -r 5ccabf34d30e -r 9fa579e2d7b9 layout/base/crashtests/crashtests.list
--- a/layout/base/crashtests/crashtests.list	Wed Nov 20 19:49:29 2024 +0000
+++ b/layout/base/crashtests/crashtests.list	Wed Nov 20 19:49:42 2024 +0000
@@ -566,3 +566,4 @@
 load 1821469.html
 load 1849898-1.html
 load 1929445.html
+asserts(1-3) load 1931933.html